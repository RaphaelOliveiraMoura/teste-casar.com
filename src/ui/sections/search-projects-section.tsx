import Image from "next/image";

export function SearchProjectsSection() {
  return (
    <section className="flex h-full flex-col items-center justify-center gap-8">
      <div className="grid place-items-center">
        <h2 className="text-xl font-semibold">
          Procure pelo Nome ou Nome de Usu치rio
        </h2>
        <p>
          Encontre os reposit칩rios de algum usu치rio digitando no campo acima
        </p>
      </div>
      <Image
        alt="Imagem de uma pessoa com uma lupa buscando por reposit칩rios na web"
        src="/undraw_people_search.svg"
        width={200}
        height={200}
      />
    </section>
  );
}
