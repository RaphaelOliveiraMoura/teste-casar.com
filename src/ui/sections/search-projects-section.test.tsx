import { render, screen } from "@testing-library/react";
import { expect, test } from "vitest";

import { SearchProjectsSection } from "./search-projects-section";

test("<SearchProjectsSection />", async () => {
  const snapshot = render(<SearchProjectsSection />);

  expect(
    screen.getByText("Procure pelo Nome ou Nome de Usu√°rio"),
  ).toBeDefined();

  expect(snapshot).toMatchSnapshot();
});
