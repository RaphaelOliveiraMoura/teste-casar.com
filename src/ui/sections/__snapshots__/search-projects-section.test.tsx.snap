// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<SearchProjectsSection /> 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="flex h-full flex-col items-center justify-center gap-8"
      >
        <div
          class="grid place-items-center"
        >
          <h2
            class="text-xl font-semibold"
          >
            Procure pelo Nome ou Nome de Usuário
          </h2>
          <p>
            Encontre os repositórios de algum usuário digitando no campo acima
          </p>
        </div>
        <img
          alt="Imagem de uma pessoa com uma lupa buscando por repositórios na web"
          data-nimg="1"
          decoding="async"
          height="200"
          loading="lazy"
          src="/undraw_people_search.svg"
          style="color: transparent;"
          width="200"
        />
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="flex h-full flex-col items-center justify-center gap-8"
    >
      <div
        class="grid place-items-center"
      >
        <h2
          class="text-xl font-semibold"
        >
          Procure pelo Nome ou Nome de Usuário
        </h2>
        <p>
          Encontre os repositórios de algum usuário digitando no campo acima
        </p>
      </div>
      <img
        alt="Imagem de uma pessoa com uma lupa buscando por repositórios na web"
        data-nimg="1"
        decoding="async"
        height="200"
        loading="lazy"
        src="/undraw_people_search.svg"
        style="color: transparent;"
        width="200"
      />
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
